distribution/docker/build.gradle

```
@@ -302,8 +288,7 @@ void addTransformDockerContextTask(Architecture architecture, DockerBase base) {
       eachFile { FileCopyDetails details ->
         if (details.name.equals("Dockerfile")) {
           filter { String filename ->
-            String result = filename.replaceAll('^RUN curl.*artifacts-no-kpi.*$', "COPY ${distributionName} /tmp/elasticsearch.tar.gz")
-            return result.replaceAll('^RUN curl.*cloudflare/zlib.*$', "COPY zlib-${cloudflareZlibVersion}.tar.gz /tmp")
+            reeturn filename.replaceAll('^RUN curl.*artifacts-no-kpi.*$', "COPY ${distributionName} /tmp/elasticsearch.tar.gz")
```

Looks like a typo here. 😊
```suggestion
            return filename.replaceAll('^RUN curl.*artifacts-no-kpi.*$', "COPY ${distributionName} /tmp/elasticsearch.tar.gz")
```